src/arithmetic/arithmetic_expression.c

```diff
@@ -53,7 +53,7 @@ SIValue AR_EXP_Evaluate(const AR_ExpNode *root, const Record r) {
                 GraphEntity *ge = (GraphEntity*)entry.ptrval;
                 SIValue *property = GraphEntity_Get_Property(ge, root->operand.variadic.entity_prop);
                 /* TODO: Handle PROPERTY_NOTFOUND. */
-                result = SI_Clone(*property);
+                result = SI_ShallowCopy(*property);
```

Comment:
What will happen in the case of PROPERTY_NOTFOUND?

Reply:
PROPERTY_NOTFOUND is just a T_NULL SIValue. So far as I can tell, we handle it just like Neo does - any comparison to it fails in a filter, it shows up as NULL in result sets, and arithmetic functions skip it.

Maybe there were issues when this TODO was added? I don't see any unexpected behaviors right now, though!